developer/build_plugin.py

```diff
@@ -454,23 +537,94 @@ def main():
     Handles argument parsing and validation and then calls the script payload.
     """
 
-    usage = "%prog source_path target_path (run with --help for more information)"
+    usage = "%prog source_path [-dskc] target_path (run with --help for more information)"
```

Comment:
The `[-dskc]` part here is a little misleading. This makes it look like flags. Maybe replace with something like `[options]` since otherwise it could get long? Or maybe just insert full option and make it long?